EnglishFilter/sections/specific.txt

```diff
@@ -623,7 +623,12 @@ hornygamer.com##center > a[onclick^="window.open("]
 ||sxyprn.com/player/p12.js
 euronews.com###js-leaderboard-top
 euronews.com##aside.o-article__aside > .OUTBRAIN
-fpo.xxx##.top
+diamondfaucet.space,fpo.xxx##.top
+diamondfaucet.space##.bottom
+diamondfaucet.space##.widget-3
+diamondfaucet.space##.widget-2
+diamondfaucet.space##.form-group > div[id^="adm-container-"]
+diamondfaucet.space##div.ablinksgroup
```

Comment:
Where is this element placed on the page?

Reply:
Take a look at the screenshots below:
![image](https://user-images.githubusercontent.com/76908410/114530368-73a57380-9c7d-11eb-8553-6f1a09ec2c65.png)
![image](https://user-images.githubusercontent.com/76908410/114530401-7a33eb00-9c7d-11eb-88c3-6b048841d022.png)
![image](https://user-images.githubusercontent.com/76908410/114530460-84ee8000-9c7d-11eb-8e5a-1c745937f727.png)
![image](https://user-images.githubusercontent.com/76908410/114530492-8d46bb00-9c7d-11eb-921f-25641d9a205f.png)
![image](https://user-images.githubusercontent.com/76908410/114530627-b23b2e00-9c7d-11eb-93c6-8cbe7e2ce0ff.png)